tutorials/algorithms/11_trotterQRTE.ipynb

```diff
@@ -0,0 +1,998 @@
+{
```

Comment:
<blockquote>First, we will create a function that takes in the parameters, and returns our Hamiltonian, as a <code>SparsePauliOp</code>.</blockquote><p> </p><p>Maybe sth like "system parameters" is a bit clearer, and I would remove the comma after Hamiltonian.</p><p> </p><p> </p><p>The introduction is very very nice, but I am missing here the introduction of the TrotterQRTE class, which the main motivation of this tutorial.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1465/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>